src/test/docker/docker-dropwizard-example/dockerBuildImage.bat

```diff
@@ -0,0 +1 @@
+docker build -t example/dropwizard:latest .
```

Comment:
Feels like repo clutter to have 8 bat/sh scripts in the repo just to run one line java and docker commands. I understand they're helpful locally, but I don't think they should be checked in.